agr_java_core/src/main/java/org/alliancegenome/core/variant/service/AlleleVariantIndexService.java

```diff
@@ -1,78 +1,78 @@
 package org.alliancegenome.core.variant.service;
 
-import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.DeserializationFeature;
-import com.fasterxml.jackson.databind.MapperFeature;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.gson.Gson;
+import static org.alliancegenome.core.config.Constants.ES_INDEX;
+
+import java.io.IOException;
+import java.util.*;
+
 import org.alliancegenome.api.entity.AlleleVariantSequence;
 import org.alliancegenome.es.util.EsClientFactory;
 import org.alliancegenome.neo4j.entity.node.*;
-import org.elasticsearch.action.search.SearchRequest;
-import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.action.search.*;
 import org.elasticsearch.client.RequestOptions;
-import org.elasticsearch.index.query.BoolQueryBuilder;
-import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.index.query.*;
 import org.elasticsearch.search.SearchHit;
 import org.elasticsearch.search.builder.SearchSourceBuilder;
 
-import java.io.IOException;
-import java.util.*;
-import java.util.stream.Collectors;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.*;
 
-import static org.alliancegenome.core.config.Constants.ES_INDEX;
+import lombok.extern.jbosslog.JBossLog;
 
+@JBossLog
 public class AlleleVariantIndexService {
-    ObjectMapper mapper=new ObjectMapper();
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    public AlleleVariantIndexService() {
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        mapper.configure(DeserializationFeature.ACCEPT_EMPTY_ARRAY_AS_NULL_OBJECT, true);
+        mapper.configure(DeserializationFeature.FAIL_ON_NULL_CREATOR_PROPERTIES, false);
+        mapper.configure(MapperFeature.DEFAULT_VIEW_INCLUSION,false);
```

Comment:
Instead of putting these options in every method its better to pull the out into the constructor considering all the methods are going to make use of the same mapper.